pkg/crc/machine/cert-renewal.go

```diff
@@ -125,12 +28,10 @@ func waitForPendingCsrs(oc oc.OcConfig) error {
 		return nil
 	}
 
-	return errors.RetryAfter(60, waitForPendingCsr, time.Second)
+	return errors.RetryAfter(60, waitForPendingCsr, time.Second*5)
```

Comment:
I'd change this in a separate commit.

Reply:
@cfergeau sure, will do that.

Reply:
How much time is needed before getting the first csr?  Less than the 4/5 minutes mentioned in the log? And once we got one CSR, isn't it very long to wait for 5 more minutes?

Reply:
@cfergeau usually second csr we get around 30sec-1min but this is not like waiting again for 5 mins but break as soon as the csr requests comes right, 5 min is a timeout in our condition.

Reply:
How long does the 1st one take to come up?
And if there is a failure, 5 minutes is a very wasteful wait if we know after 2 minutes that it's never going to succeed.

Reply:
> How long does the 1st one take to come up?

@cfergeau first one does take around 4-5 mins, didn't record exact time but around that on my testing instance.

Reply:
If first CSR request takes 4/5 minutes to appear, and since this time is dependant on the machine running crc, isn't a 5 minutes timeout possibly going to be a bit too short for the 1st request? Might be better to add a `timeout` arg to `waitForPendingCsrs, and wait for 8 to 10 minutes (or is it too much?) for the first one, and 2 minutes for the second one.

Reply:
@cfergeau yesterday when @anjannath trying this PR the second CSR also took around 4min to appear so I am not sure if we put different timeout for different csr requests if you want to increase the first one to 8-10 mins then I would also wait around same time for second csr.